GDJS/Runtime/events-tools/objecttools.ts

```diff
@@ -651,5 +652,13 @@ namespace gdjs {
       );
       this.callbacks.delete(runtimeObject);
     }
+
+    getLocalVariablesContainers(): Array<gdjs.VariablesContainer> {
+      return this.localVariablesContainers;
+    }
+
+    setLocalVariablesContainers(variablesContainers: Array<gdjs.VariablesContainer>): void {
+      gdjs.copyArray(variablesContainers, this.localVariablesContainers);
+    }
```

Comment:
Let's be verbose so that searching this is obvious in the code:

getLocalVariablesContainer => `getBackedUpLocalVariablesContainer` 
setLocalVariablesConainer => sounds like a setter. Instead `backupLocalVariablesContainers` (the caller will understand that a "heavy operation" is involved)